### PR TITLE
Dpys

### PR DESCRIPTION
**Pull request by dPys**

### CODE REVIEW

File: ndmg/graph/gen_graph.py [hunk 1]
```diff
@@ -196,7 +196,7 @@ def make_graph(self, error_margin=2, overlap_thr=1, voxel_size=2):
         mx = len(np.unique(self.rois.astype(np.int64))) - 1
         self.g = nx.Graph(ecount=0, vcount=mx)
         edge_dict = defaultdict(int)
-        node_dict = dict(zip(np.unique(self.rois) + 1, np.arange(mx) + 1))
+        node_dict = dict(zip(np.unique(self.rois).astype('int16') + 1, np.arange(mx) + 1))
```
Comment (loftusa) [inline ndmg/graph/gen_graph.py:199]: Why do we need this to be `int16`?

File: ndmg/register/gen_reg.py [hunk 2]
```diff
@@ -657,7 +657,7 @@ def atlas2t1w2dwi_align(self, atlas, dsn=True):
             self.atlas_img = nib.load(self.dwi_aligned_atlas)
         else:
             self.atlas_img = nib.load(self.aligned_atlas_t1mni)
-        self.atlas_data = self.atlas_img.get_data().astype("int")
+        self.atlas_data = np.around(self.atlas_img.get_data()).astype("int16")
```
Comment (loftusa) [inline ndmg/register/gen_reg.py:660]: Why np.around now?

**Review by loftusa — CHANGES_REQUESTED** [review]
Looked through -- looks like the only changes are switching to `int16` and using `np.around`? What's the backstory / reasoning?

File: ndmg/graph/gen_graph.py (hunk 1 above)
Comment (dPys) [reply ndmg/graph/gen_graph.py:199]: consistency. Not all parcellations/atlases use integer intensities, and this imposes that as a standard.

File: ndmg/register/gen_reg.py (hunk 2 above)
Comment (dPys) [reply ndmg/register/gen_reg.py:660]: In this case, rounding ensures that atlas intensities of 0.999 are interpreted as 1 and not 0.

**Comment by dPys** [comment]
> Looked through -- looks like the only changes are switching to `int16` and using `np.around`? What's the backstory / reasoning?

Imposes consistency of numeric datatypes for parcellations/atlases (not all of which are always created equal)

**Review by loftusa — APPROVED** [review]

**Pull request merged**